content/blog/2020-07-09-cloud-update/index.Rmarkdown

```diff
@@ -0,0 +1,128 @@
+knitr::include_graphics("https://mine-cetinkaya-rundel.github.io/rstudio-cloud-webinar/img/rstudio-ide.png")
+```
+
+RStudio Cloud moves the IDE into your web browser instead:
+
+```{r echo=FALSE, out.width="75%", fig.cap="Cloud screenshot from Dr. Mine Çetinkaya-Rundel"}
+knitr::include_graphics("https://mine-cetinkaya-rundel.github.io/rstudio-cloud-webinar/img/rstudio-cloud.png")
+```
+
+This makes [RStudio Cloud](https://rstudio.cloud) a lightweight way for anyone to do, share, teach, and learn data science online. For educators, it simplifies the process of teaching R to your students or colleagues by letting the instructor focus on the content, not the infrastructure. Students learn directly within their web browsers, with nothing to install locally, and with no infrastructure for the instructor to maintain.
+
+RStudio Cloud is designed from the ground up to make data science teaching easier for instructors and students:
+
+-   Classes can be organized in workspaces
+-   Members can be assigned different roles: instructor, TA, student
+-   Projects can be public or private
+-   Projects can be made into assignments
+-   Students can make copies of projects created by the instructor, with the necessary environment automatically replicated.
+-   Instructor can peek into and edit student projects
+
+In addition to teaching data science, RStudio Cloud is a great platform for doing data science as well. The ability to share projects makes it easy for researchers from different groups or institutions to collaborate, and is useful for any data science team that wants to build and share data science work, without having to maintain their own IT infrastructure.
+
+To learn more:
+
+-   Visit <https://rstudio.com/products/cloud>,
+-   Check out our recent webinar on [Teaching R Online with RStudio Cloud](https://rstudio.com/resources/webinars/teaching-r-online-with-rstudio-cloud/),
+-   Sign up at <https://rstudio.cloud/> and get started for free.
+
+## What's changing with RStudio Cloud?
+
+After an extensive alpha/beta period, RStudio Cloud will be generally available for use and purchase in the fall. (Until then, you can still sign up for the Cloud Free plan - the usage limits are currently not enforced). While RStudio Cloud will always offer a free plan for individuals, beginning August 3, 2020, we will also offer paid premium plans for individuals, instructors, students and organizations. Learn more on our [Plans & Pricing page](https://rstudio.cloud/plans/).
+
+## Why are we making this change?
+
+At RStudio, we offer our professional products ([RStudio Server Pro](https://rstudio.com/products/rstudio-server-pro/), [RStudio Connect](https://rstudio.com/products/connect/), [RStudio Package Manager](https://rstudio.com/products/package-manager/)) for free for instructional use. You can read more about these options [here](/teach/tools/#computing-infrastructure).
```

Comment:
I'd reframe this to something like:

At RStudio, we're committed to supporting educators using R, that's why we offer [our](https://rstudio.com/products/rstudio-server-pro/) [professional](https://rstudio.com/products/connect/) [products](https://rstudio.com/products/package-manager/) for free. We'd love to also make RStudio.cloud free for everyone, but unfortunately RStudio.cloud is fundamentally different because it's not just software, but is also computing. During the beta program we provided 3.5 million hours of compute time for free, and unfortunately we just can't afford to keep doing that — we need to need to cover our ongoing hosting costs so that we can continue to offer the service. In short: the fees help us offset the cost of operating the service, and enable us to continue to provide a free plan for individuals.

Reply:
(Needs a decent amount of word smithing but you get the idea)